### ISSUE TITLE
Some questions about the selection of the training loss of a model

### ISSUE THREAD

**Issue by qq1332427275** [LAION-AI/CLAP#152]
Hello Author! Your project is very meaningful, I would like to do a deep research on it, but at present there is a confusion that has not been solved, in the model training of the loss selection, I see that your code uses the cross-entropy loss, but I see that the CLIP model training uses the contrast loss (the loss of the two directions summed and averaged) What is the difference between these two ways? I have been confused about it and I hope the author can help me out, thanks a lot for reading! Looking forward to your reply!

**Comment by lukewys** [comment]
Hi, the infoNCE loss is implemented as cross-entropy loss. Please see the original CLIP paper for the derivation.
